src/main/scala/org/broadinstitute/dsde/rawls/RawlsException.scala

```diff
@@ -7,7 +8,8 @@ class RawlsException(message: String = null, cause: Throwable = null) extends Ex
 /**
  * Specifies which HTTP status code to return if/when this exception gets bubbled up.
  */
-class RawlsExceptionWithStatusCode(message: String = null, cause: Throwable = null,
-  statusCode: StatusCode = StatusCodes.InternalServerError) extends RawlsException(message, cause) {
+class RawlsExceptionWithErrorReport(message: String = null, cause: Throwable = null,
+  statusCode: StatusCode = StatusCodes.InternalServerError, errorReport: ErrorReport = null) extends RawlsException(message, cause) {
```

Comment:
I don't think this makes sense to have a message and error report should never be null, the message on the parent exception should be errorReport.message